<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinGameWindow</class>
 <widget class="QDialog" name="JoinGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dołącz do gry</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dołącz do istniejącego pokoju</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>351</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbl1">
      <property name="text">
       <string>Podaj swój pseudonim</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nickLineEdit">
      <property name="maxLength">
       <number>16</number>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl2">
      <property name="text">
       <string>Kod dostępu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="codeLineEdit">
      <property name="maxLength">
       <number>4</number>
      </property>
      <property name="placeholderText">
       <string>np. 1234</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="planningLogo">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>221</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/4.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="joinBtn">
      <property name="text">
       <string>Dołącz</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backBtn">
      <property name="text">
       <string>Wróć do menu</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
